src/tribler/core/components/libtorrent/download_manager/download.py

```diff
@@ -318,26 +322,23 @@ def on_tracker_error_alert(self, alert: lt.tracker_error_alert):
         """
         # The try-except block is added as a workaround to suppress UnicodeDecodeError in `repr(alert)`,
         # `alert.url` and `alert.msg`. See https://github.com/arvidn/libtorrent/issues/143
-        try:
-            self._logger.error(f'On tracker error alert: {alert}')
-            url = alert.url
-
-            if alert.msg:
-                status = 'Error: ' + alert.msg
-            elif alert.status_code > 0:
-                status = 'HTTP status code %d' % alert.status_code
-            elif alert.status_code == 0:
-                status = 'Timeout'
-            else:
-                status = 'Not working'
+        self._logger.error(f'On tracker error alert: {safe_repr(alert)}')
+        url = alert.url
+
+        if alert.msg:
+            status = 'Error: ' + alert.msg
```

Comment:
I believe this line should be tested since some portions of this function have already been covered by tests.

Reply:
I added more tests to cover all branches of this function